### ISSUE TITLE
When should we rotate our structures?

### ISSUE THREAD

**Issue by pc494** [pyxem/diffsims#59]
**Describe the bug**
`simulate_rotated_structure()`, in theory, "rotates" a diffpy lattice. In practice ... it's not that simple.

**To Reproduce**

Make a boring orthorhombic lattice. 
Make a second "rotated" lattice - the rotation here is a cut down version of what we do in `simulate_rotated_structure()`
```
canonical = diffpy.structure.lattice.Lattice(4,2,1, 90, 90, 90)
R = np.asarray([[1/np.sqrt(2),1/np.sqrt(2),0],[-1/np.sqrt(2),1/np.sqrt(2),0],[0,0,1]])
halfhalf  = diffpy.structure.lattice.Lattice(base=(R@canonical.base))
```

The problem is that the `halfhalf` lattice has two cell parameters of `3.162...` ~ this makes life hard work.

**Solution**
Cut all the rotation code that involved diffpy, spit cartesians out of `get_points_sphere()` then 
manually rotate the cartesians with a rotation matrix. I actually think this might be quite simple.
`
get_kinematical_intensities` might also need a revisit

**Comment by pc494** [comment]
Taking from @EirikOpheim's example - the implementation goes

```
from transforms3d.euler import euler2mat

### Case 1
cartesian_coordinates = np.asarray([[1,0,0],[0,1,1]])
rotation = (60,60,0)
ai,aj,ak = np.deg2rad(rotation[0]),np.deg2rad(rotation[1]),np.deg2rad(rotation[2])
R = euler2mat(ai,aj,ak,axes='rzxz')
cartesian_coordinates = np.matmul(R,cartesian_coordinates.T).T

#### Case 2
cartesian_coordinates = np.asarray([[1,0,0],[0,1,1]])
rotation = (30,60,0)
ai,aj,ak = np.deg2rad(rotation[0]),np.deg2rad(rotation[1]),np.deg2rad(rotation[2])
R = euler2mat(ai,aj,ak,axes='rzxz')
cartesian_coordinates = np.matmul(R,cartesian_coordinates.T).T
```

**Comment by pc494** [comment]
This was resolved by #60
